README.md

```diff
@@ -108,17 +108,27 @@ $ yarn json2xlsx options [<sheet name> ...]
 $ yarn xlsx2json options [<sheet name> ...]
```

Comment:
Remove `[<sheet name> ...]` from line above since `json2xlsx` does not suport anything other than options.